src/LiveQuery/ParseLiveQueryServer.ts

```
@@ -1056,6 +1081,192 @@ class ParseLiveQueryServer {
+      }
+      return [];
+    }
+    const subObject = object[path[0]];
+    if (!subObject) {
+      return [];
+    }
+    return this.findPointers(subObject, path.slice(1));
+  }
+
+  replacePointers(object: any, path: Array<string>, replace: any): any {
+    if (object instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)